### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#313] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-09 172614](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/5aaae898-5c47-4b66-8de3-e4b03b8e5d54)

**Comment by Elia-Belli** [comment]
Siccome $\phi:G\rightarrow G'$ è un isomorfismo allora vale $\phi(g_1\star g_2)=\phi(g_1)\star \phi(g_2)$ e $\phi$ è biettiva:
L'inversa di una funzione biettiva è a sua volta biettiva $\Rightarrow \phi^{-1}:G'\rightarrow G$ biettiva.

Rimane da dimostrare che vale $\phi^{-1}(y_1\star y_2)=\phi^{-1}(y_1)\star \phi^{-1}(y_2),\forall y_1,y_2 \in G'$:
Poniamo per assurdo $\phi^{-1}(y_1\star y_2)\neq\phi^{-1}(y_1)\star \phi^{-1}(y_2)$ con $g_1,g_2$ rispettivamente preimmagini di $y_1,y_2$
$\phi(g_1\star g_2)=y_1\star y_2\Rightarrow \phi^{-1}(y_1\star y_2)=g_1\star g_2$
$\phi(g_1)\star \phi(g_2)=y_1\star y_2\Rightarrow \phi^{-1}(y_1)\star \phi^{-1}(y_2)=g_1\star g_2$
ma $g_1\star g_2\neq g_1\star g_2$ assurdo $\Rightarrow \phi^{-1}(y_1\star y_2)=\phi^{-1}(y_1)\star \phi^{-1}(y_2)\Rightarrow \phi$ è un isomorfismo

**Comment by daromacarlo** [comment]
![Immagine 2024-01-06 163101](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/27e2e742-51d3-4d30-b7c8-d763fb2c9415)
![Immagine 2024-01-06 163114](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/73dc59ce-bd75-47b8-bda1-d2b4c448f363)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/245f2b9f-66f6-4344-b875-d6684c011fce)
